code/modules/research/designs/weapons.dm

```diff
@@ -212,9 +212,9 @@
 	id = "smg"
 	req_tech = list("combat" = 4, "materials" = 3)
 	build_type = PROTOLATHE
-	materials = list(MAT_IRON = 8000, MAT_SILVER = 2000, MAT_DIAMOND = 1000)
+	materials = list(MAT_IRON = 10000, MAT_GLASS = 5000)
```

Comment:
Security has the armory or cargo for all those situations - Research shouldn't really be designed with them in mind.